pages/SavingsTable.js

```diff
@@ -0,0 +1,29 @@
+import BasePage from './BasePage';
+
+class SavingsTable extends BasePage {
```

Comment:
You can follow the same logic than with the account creation stuff here (Both savings-view and checking-view share most user interface elements), therefore creating a generic AccountsTable, and then make SavingsTable and CheckingTable inherit from it, as well as placing logic they share there, so savings and checking specific pages only have to implement what is unique to them.

For what I can tell, most logic about the table is completely shared. Only thing that changes is part of the xpath selector (the part that selects based on a specific test that individualizes the type of account you are selecting), the rest is completely shared